flik/linesearch/test/test_conditions.py

```diff
@@ -1 +1,35 @@
 """Tests for flik.linesearch.conditions."""
+
+import numpy as np
+from nose.tools import assert_raises
+
+from conditions import check_armijo
+
+def test_armijo_pars():
```

Comment:
I think there can be just one test for testing the armijo method, rather than one for testing its output and another for checking its parameters.